### ISSUE TITLE
Linting fails with "@angular-eslint/eslint-plugin-template currently only supports 1 Component per file" after upgrading to alpha.28

### ISSUE THREAD

**Issue by orestisioakeimidis** [angular-eslint/angular-eslint#73]
Linting suddenly fails after upgrading to alpha.28 with `@angular-eslint/eslint-plugin-template currently only supports 1 Component per file`.

I am suspecting something here https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/processors.ts#L46.

This `componentDecoratorNodes.push(decorator);` command might need to go inside the curly brackets.

**Comment by BO41** [comment]
The empty `if` seems definitely wrong :smile:

**Comment by JamesHenry** [comment]
Oh boy, that's a first 😅 

@orestisioakeimidis I'm guessing you have a particular file with multiple classes with decorators in? That case is not represented in the integration tests which is why this was not caught.

I will add something which reproduces this at the same time as the fix

**Comment by orestisioakeimidis** [comment]
@JamesHenry No I don't have a file with multiple classes with decorators, but I have a class with multiple decorators, which is the reason that linting is failing, I just tested it.

```
@UntilDestroy()
@Component({
  ...
})
export class MyComp {}
```

**Comment by JamesHenry** [comment]
Fixed in 0.0.1-alpha.29
